chrome/content/zotero/xpcom/data/feedItems.js

```diff
@@ -94,6 +94,35 @@ Zotero.FeedItems = new Proxy(function() {
 		return this.getAsync(id);
 	});
 	
+	this.toggleReadByID = Zotero.Promise.coroutine(function* (ids, state) {
+		if (!Array.isArray(ids)) {
+			if (typeof ids != 'string') throw new Error('ids must be a string or array in Zotero.FeedItems.toggleReadByID');
+			
+			ids = [ids];
+		}
+		let items = yield this.getAsync(ids);
+		
+		if (state == undefined) {
```

Comment:
Er, scratch that: `false` != `undefined`. (It's `null` == `undefined`, and `!state` would be true for both `false` and `undefined`. But this will be false for `false`.)